dist/index.js

```diff
@@ -343,13 +343,12 @@ ${chunk.changes
           });
           const comments = yield analyzeCode(filteredDiff, prDetails);
           if (comments.length > 0) {
-            // await createReviewComment(
-            //   prDetails.owner,
-            //   prDetails.repo,
-            //   prDetails.pull_number,
-            //   comments
-            // );
-            yield console.log("Comments:", comments);
+            yield createReviewComment(
```

Comment:
You meant to change this to yield?

Reply:
It looks weird because this is the transpiled code (from Typescript to Javascript). The await instruction in Typescript gets turned that way into some `yield` instruction. It is better to compare with the Typescript source code down below, in the `main.ts` file.